media-sound/dvda-author/dvda-author-10.06-r1.ebuild

```diff
@@ -36,16 +40,3 @@ src_configure() {
 		--with-config=/etc \
 		$(use_with debug debug full)
 }
-
-src_compile() {
-	# make[1]: warning: jobserver unavailable: using -j1.
-	#                   Add '+' to parent make rule.
-	emake -j1
-}
-
-src_install() {
-	newbin src/dvda ${PN}
-	insinto /etc
-	doins ${PN}.conf
-	dodoc AUTHORS BUGS ChangeLog EXAMPLES HOWTO.conf LIMITATIONS NEWS TODO
-}
```

Comment:
`src_install` works just fine with `default_src_prepare`?

Reply:
Yep, we do have a bit more files installed with default src_install